framework/src/Volo.Abp.Localization/Volo.Abp.Localization.csproj

```diff
@@ -29,4 +29,11 @@
     <ProjectReference Include="..\Volo.Abp.VirtualFileSystem\Volo.Abp.VirtualFileSystem.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Fody" Version="6.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
```

Comment:
We can remove these because it is already configured in `configureawait.props`.

Reply:
Resolved apologies